buildSrc/src/main/java/Dependencies.kt

```diff
@@ -3,16 +3,16 @@ const val android_min_sdk_version = 21
 const val android_target_sdk_version = 32
 
 // Project config
-const val kotlin_version = "1.6.21"
+const val kotlin_version = "1.7.10"
 const val android_gradle_plugin_version = "7.2.2"
 const val ktlint_version = "10.2.0"
 
 // KMM
-const val ktor_version = "1.6.8"
-const val serialization_version = "1.3.2"
+const val ktor_version = "2.0.3"
+const val serialization_version = "1.3.3"
 const val coroutines_version = "1.6.3-native-mt"
 const val sql_delight_version = "1.5.3"
-const val kotlinx_datetime_version = "0.3.2"
+const val kotlinx_datetime_version = "0.4.0"
 const val uuid_lib_version = "0.3.1"
 const val isolate_version = "1.2.0"
```

Comment:
Remove it as you already remove the dependency in the code.

Reply:
Removed